src/main/java/frc/robot/subsystems/ArmSystem.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.subsystems;
+
+import com.revrobotics.*;
+import edu.wpi.first.wpilibj.DutyCycleEncoder;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.RobotMap;
+
+public class ArmSystem extends SubsystemBase {
+    private final CANSparkMax motor;
+    private final RelativeEncoder neoEncoder;
+    private final DutyCycleEncoder dutyCycleEncoder;
+
+    public ArmSystem() {
+        motor = new CANSparkMax(RobotMap.ARM_MOTOR_PORT, CANSparkLowLevel.MotorType.kBrushless);
+        neoEncoder = motor.getEncoder();
+        dutyCycleEncoder = new DutyCycleEncoder(RobotMap.ARM_ENCODER_PORT);
+
+        motor.restoreFactoryDefaults();
+
+        neoEncoder.setPosition(dutyCycleEncoder.getAbsolutePosition() - RobotMap.ABSOLUTE_ENCODER_ZERO_OFFSET);
```

Comment:
To make this native motor rotations you must consider the gear ration too. Note that the absolute encoder is _after_ the gear box and the motor is _before_ it,

Reply:
done

Reply:
Dividing converts from motor rotations to post gear box. As the ratio describes amount of motor rotations to post gear box rotations.

So we need to multiply to convert from post gear box to motor rotations

Reply:
done